Source/Core/Core/HW/Memmap.cpp

```diff
@@ -405,7 +405,7 @@ u8* MemoryManager::GetPointerForRange(u32 address, size_t size) const
 
   if (span.data() == nullptr)
   {
-    // The address isn't in a valid memmory region.
+    // The address isn't in a valid memory region.
```

Comment:
Do you mind moving this to the first commit?^^;

Reply:
Oh, whoops